### PR TITLE
chore: use devinfra assistant PAT in VHD automation + abe2e pipelines

### PR DESCRIPTION
**Pull request by cameronmeissner**
**What type of PR is this?**
updates VHD automation pipeline to use dev infra's "devinfraassistant" service account for downloading artifacts from ADO, instead of personal accounts

TODO: investigate how to do the same for github PATs?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by anujmaheshwari1 — APPROVED** [review]
lgtm, thanks for updating this!

File: .pipelines/.vsts-vhd-windows-automation.yaml [hunk 1]
```diff
@@ -16,13 +16,16 @@ parameters:
...
+variables:
+  - group: "AKS Dev Assistant (KV)"
+
 steps:
 - bash: |
         az extension add -n azure-devops
         echo $MAPPED_ADO_PAT | az devops login --organization=https://dev.azure.com/msazure
         az devops configure --defaults organization=https://dev.azure.com/msazure project=CloudNativeCompute
   env:
-        MAPPED_ADO_PAT: $(ADO_PAT)
```
Comment (djsly) [inline .pipelines/.vsts-vhd-windows-automation.yaml:25]: where is this one defined ? should it be removed from the pipeline var ? or from the other KV group ?

**Comment by djsly** [comment]
What about the Linux pipeline and the e2e ? should those two logic also be updated to not use the ADO_PAT ?

File: .pipelines/.vsts-vhd-windows-automation.yaml (hunk 1 above)
Comment (cameronmeissner) [reply .pipelines/.vsts-vhd-windows-automation.yaml]: yes, we will remove it from the pipeline vars once the changes are made

**Comment by cameronmeissner** [comment]
> What about the Linux pipeline and the e2e ? should those two logic also be updated to not use the ADO_PAT ?

actually originally updated the wrong file, now updating e2e + linux automation pipelines

**Pull request merged**